dist/windows/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 install(FILES qt.conf
-    DESTINATION ${CMAKE_INSTALL_BINDIR}
+    DESTINATION "${PROJECT_NAME}"
```

Comment:
I'm not familiar with CMake subtleties... Where is it supposed to be installed now?

Reply:
@glassez 

It will be installed in the same directory as the executable, same as previously.

We don't need the extra `bin` folder in the path; following the standard Unix FHS conventions doesn't make sense here. 

But thanks for pointing this out, I realize now that I had it right the first time in https://github.com/glassez/qBittorrent/pull/1, the DESTINATION should be `"."`, not `"${PROJECT_NAME}"`. Otherwise `cmake --install build` will install files to `C:/Program Files/qBittorrent/qBittorrent` (creating an unnecessary extra `qBittorrent` directory), because `CMAKE_INSTALL_PREFIX` (https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html?highlight=cmake_install_prefix) is set to `"C:/Program Files/${PROJECT_NAME}"` by default on Windows.

Reply:
@glassez The user can already achieve this in the standard way: either by setting `-DCMAKE_INSTALL_PREFIX=C:/some/dir/qBittorrent/bin` at configure-time or by passing `--prefix C:/some/dir/qBittorrent/bin` to the invocation of `cmake --install`.